### ISSUE TITLE
[BE] Integration of AI into the backend

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#30]
Subsequent issues: #31 #32 

- research on storage of AI model

## Acceptance criteria
- [x] text can be inserted from the console
- [x] response is printed in the console

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
